book/src/future_topics.md

```diff
@@ -9,9 +9,10 @@ jni-rs has additional abstractions that haven't been discussed, like executors.
 JNI also has a few features that need their own sections (critical APIs, and
 local and global references).
 
-There are also performance optimizations that haven't been covered. It is more
-efficient to issue calls from Java to native code, than it is to go from native
-code to Java, and it's possible to cache class/methodIds/fieldIds.
+There are also performance optimizations that haven't been covered. For example,
+it is more efficient to issue calls from Java to native code than from native
+code to Java, and caching class/methodIds/fieldIds can reduce the number of
```

Comment:
Possibly, this section could be useful to link: https://docs.rs/jni/0.18.0/jni/struct.JNIEnv.html#checked-and-unchecked-methods